lib/registries/addon/branded/discover/styles.scss

```diff
@@ -0,0 +1,5 @@
+.PrimaryColor {
+    background-color: var(--primary-color);
+    height: 200px;
+    width: 200px;
+}
```

Comment:
Will get removed in the next PR. This is here to demonstrate a proof of concept with the branding/css variables working